code/_onclick/telekinesis.dm

```diff
@@ -144,8 +144,8 @@ var/const/tk_maxrange = 15
 		apply_focus_overlay()
 		focus.throw_at(target, 10, 1,user)
 		last_throw = world.time
-		user.changeNext_move(CLICK_CD_MELEE)
```

Comment:
I'd suggest putting the click delay back, unless there is a very good reason to remove it